### ISSUE TITLE
Has `AccessibilityInfo` Error when build ios

### ISSUE THREAD

**Issue by minsooshin** [aksonov/react-native-router-flux#1903]
### Version
Tell us which versions you are using: 

- react-native-router-flux v3.39.1
- react-native v0.44.2
- react v16.0.0-alpha.6

### Expected behaviour

Build successfully.

### Actual behaviour

Go error
```sh
error: bundling: UnableToResolveError: Unable to resolve module `AccessibilityInfo` from `/Users/mshin/Workspace/study/react-native/manager/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module does not exist in the module map or in these directories:
  /Users/mshin/Workspace/study/react-native/manager/node_modules/react-native/node_modules
,   /Users/mshin/Workspace/study/react-native/manager/node_modules

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset packager cache: `rm -fr $TMPDIR/react-*` or `npm start -- --reset-cache`.
    at UnableToResolveError (/Users/mshin/Workspace/study/react-native/manager/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:525:5)
    at ResolutionRequest._resolveNodeDependency (/Users/mshin/Workspace/study/react-native/manager/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:394:11)
    at tryResolveSync (/Users/mshin/Workspace/study/react-native/manager/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:136:20)
    at tryResolveSync (/Users/mshin/Workspace/study/react-native/manager/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:66:12)
    at ResolutionRequest.resolveDependency (/Users/mshin/Workspace/study/react-native/manager/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:134:22)
    at dependencyNames.map.name (/Users/mshin/Workspace/study/react-native/manager/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:145:59)
    at Array.map (native)
    at ResolutionRequest.resolveModuleDependencies (/Users/mshin/Workspace/study/react-native/manager/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:145:42)
    at Promise.resolve.then (/Users/mshin/Workspace/study/react-native/manager/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:169:21)
    at process._tickCallback (internal/process/next_tick.js:103:7)
Bundling `index.ios.js`  61.1% (129/165), failed.
```

### Steps to reproduce

1. Install by `yarn add react-native-router-flux`
2. `react-native run-ios`

### Source code
```js
// App.js
import React, { Component } from 'react';
import { View, StatusBar } from 'react-native';
import { Provider } from 'react-redux';
import { createStore, applyMiddleware } from 'redux';
import thunk from 'redux-thunk';
import firebase from 'firebase';
import reducers from './reducers';
import Router from './Router';

class App extends Component {
  componentWillMount() {
    firebase.initializeApp({
      ...
    });
  }
  render() {
    const store = createStore(reducers, {}, applyMiddleware(thunk));
    return (
      <Provider store={store}>
        <View>
          <StatusBar barStyle="dark-content" />
          <Router />
        </View>
      </Provider>
    );
  }
}

export default App;
```

```js
// Router.js
import React from 'react';
import { Scene, Router } from 'react-native-router-flux';
import LoginForm from './components/LoginForm';

const RouterComponent = () => {
  return (
    <Router>
      <Scene key="login" component={LoginForm} title="Login" />
    </Router>
  );
};

export default RouterComponent;
```

**Comment by Kennytian** [comment]
same issue +1

**Comment by Kennytian** [comment]
same issue other guys https://github.com/facebook/react-native/issues/14209

**Comment by arabscimitar** [comment]
same issue on android

**Comment by Irosshipaz** [comment]
Very same on Android, and it doesn't let me continue a paid course...
Hate when things like this happen.

**Comment by killinit** [comment]
Same issue

**Comment by milesaylward** [comment]
Got mine to work some research. Ended up having to downgrade my react-native and use a specific version of RNRF. Delete your node modules folder. Paste the code below into your package.json and npm install everything. Make sure you close your simulator and additional terminal windows. 
```
{
  "name": "manager",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "start": "node node_modules/react-native/local-cli/cli.js start",
    "test": "jest"
  },
  "dependencies": {
    "react": "16.0.0-alpha.6",
    "react-native": "0.43.0",
    "react-native-router-flux": "3.38.0",
    "react-redux": "^5.0.4",
    "redux": "^3.6.0",
    "redux-thunk": "^2.2.0",
    "firebase": "^3.9.0"
  },
  "devDependencies": {
    "babel-jest": "20.0.0",
    "babel-preset-react-native": "1.9.1",
    "eslint-config-rallycoding": "^3.2.0",
    "jest": "20.0.0",
    "react-test-renderer": "16.0.0-alpha.6"
  },
  "jest": {
    "preset": "react-native"
  }
}
```

**Comment by hellomaya** [comment]
I have met the same issue, but I resolve it, by finding that the issue is all because, that RNRF 0.39.1 explicitly depend on react-native 0.44.0, in the package.json file, if you update it with 0.44.2, then npm won't install twice different version react native package (one is 0.44.0, one is 0.44.2), thus everything is fine on my side.

**Comment by compojoom** [comment]
I'm running into the same issue. 
In my case I'm trying to upgrade to 0.46 alpha2. 

The second I update RNRF to version 3.39.1 I run into the AccessibilityInfo error

**Comment by hellomaya** [comment]
@compojoom 

Try 3.40.1, it probably work, as it will not depend on react-native explicitly.

**Comment by compojoom** [comment]
@hellomaya - thanks Tom. it seems to work with 3.40.1. - now if I could also get other packages to work....

Why is not 3.40.1 listed under releases?

**Comment by hellomaya** [comment]
@compojoom 

good to know it works, I really don't know why, but I got to have same problem, and tried 3.40.1 lately, from my reading, the code is much better compatible with latest react/react-native. I hope they will release soon.

**Comment by bernardakl** [comment]
Many issues with react and few have solutions!

**Comment by compojoom** [comment]
The above error is really strange and I'm starting to believe that it doesn't have anything to do with router flux or other components. I got this error and several other similar errors yesterday when I tried upgrading my project to 0.46.4

This thing costed me like 2h to resolve, but I have no idea what fixed it. I tried cleaning caches, updating packages and I was always presented with such an error screen. I killed node. Nothing. Than as last resort - I restarted the computer and it started working...

**Comment by bernardakl** [comment]
No it's probably a react-native-router-flux issue, 
when i downgraded to version 3.38.0 problem was fixed.

**Comment by compojoom** [comment]
well, I'm running 0.46.4 with rnrf 4.0.alpha12...

**Comment by itinance** [comment]
i can confirm that this issue re-appears on iOS with 4.0.0-beta12 but has gone with all other 4.0.0_beta-versions before

On Android the beta12 is required for being able to draw the drawer-menuicon, but on iOS this "unresolved"-error occurs

**Comment by jaimecorrea** [comment]
Same issue here after upgrade RNRF from 3.40.1 to 4.0.0-beta12

**Comment by aksonov** [comment]
Sorry, RNRF doesn't use`AccessibilityInfo`. Clear cache, etc.

**Comment by huhu521241** [comment]
delete you node_module and npm install    then success

**Comment by Symous** [comment]
Just try to restart your packager or your computer...
